### DISCUSSION TITLE
Will it be possible to run this project on Windows in the future?

### DISCUSSION THREAD

**Discussion by runhuayao** [mikel-brostrom/yolo_tracking#898] [Ideas]
Although this may make me appear ignorant, I truly hope that it will be possible to execute the yolov8_tracking project in Windows in the future. When I attempted to run the project in Windows, the result that popped up was:
The following layers are discarded due to unmatched keys or layer size: ['classifier.weight', 'classifier.bias']

**Comment by mikel-brostrom** [comment]
That is the regular dehavior. Some layers are popped from the classifiers in order to use them as feature extractors.

**Reply by runhuayao** [reply]
Thank you very much for your answer. I understand now.
